rpcs3/Emu/RSX/Overlays/overlay_perf_metrics.cpp

```diff
@@ -459,10 +465,15 @@ namespace rsx
 						[[fallthrough]];
 					}
 					case detail_level::minimal:
+					{
+						[[fallthrough]];
```

Comment:
No compiler will warn about empty code fallback, FYI.

Reply:
I thought it was just to emphasize it so that an overzealous dev doesn't add some break;